src/renderer/commands/drawObject.ts

```diff
@@ -64,7 +66,7 @@ export function drawObject(
         frag: `
             precision mediump float;
 
-            const int MAX_LIGHTS = 1; // TODO(davepagurek) [#10] Increase this to allow more lights
+            const int MAX_LIGHTS = ${regl.prop('maxLights')};
```

Comment:
This is currently giving me an error. I'm a bit unfamiliar with `regl`, so I'd appreciate any help @davepagurek.

<img width="1279" alt="screen shot 2018-05-02 at 4 25 46 pm" src="https://user-images.githubusercontent.com/8942499/39547422-8b3a4efc-4e25-11e8-8fb2-742ac42fd7d9.png">

Reply:
This is because `regl.prop('maxLights')` returns a **function**, not a value. I think you just want `maxLights` directly since it's passed in at the shader's compile time

Reply:
so actually I think keeping `maxLights` as a second argument in addition to a regl instance is good